default_launch.json

```diff
@@ -8,7 +8,13 @@
     "dimensions": {
       "x_meters": 1500,
       "z_meters": 1500
-    }
+    },
+    "legacy_flags": [
```

Comment:
What's the purpose of this flag?

Reply:
See https://docs.improbable.io/reference/14.4/shared/project-layout/launch-config#legacy-flags. Basically, if not using FPL, you need to set Runtime flags this way.

Reply:
OK, fair enough!